src/AccessibilityInsights.SharedUx/Utilities/HelperMethods.cs

```diff
@@ -50,6 +50,21 @@ public static Visibility GeneralFileBugVisibility
             }
         }
 
+        /// <summary>
+        /// Get a "File Issue" column width that is appropriate for the currently configured font size
+        /// </summary>
+        public static double FileIssueColumnWidth
+        {
+            get
+            {
+                switch (ConfigurationManager.GetDefaultInstance().AppConfig.FontSize)
+                {
+                    case Enums.FontSize.Small: return 80.0;
+                    default: return 90.0;
```

Comment:
>default [](start = 20, length = 7)

I used a switch instead of an if as defensive programming about future text sizes being added. Happy to switch it if preferred